test/runtests.jl

```diff
@@ -36,7 +36,12 @@ import BioSequences:
         @test FASTA.sequence(record, 2:3) == dna"CG"
         @test FASTA.sequence(String, record) == "ACGT"
         @test FASTA.sequence(String, record, 2:3) == "CG"
-        @test record == FASTA.Record(">foo\nACGT\n")
+
+        record1 = FASTA.Record("id", "desc", "AGCT")
+        record2 = FASTA.Record("id", "desc", "AGCT")
```

Comment:
Can you also add some records that lack some fields (esp description), or have differences, and add tests that check for inequality?